.kibana-plugin-helpers.json

```diff
@@ -1,11 +1,12 @@
 {
-  "buildSourcePatterns": [
+  "serverSourcePatterns": [
     "package.json",
     "yarn.lock",
     "tsconfig.json",
-    "{public,server}/**/*",
-    "kibana.json",
+    "common/**/*",
+    "public/**/*",
+    "server/**/*",
     "target/**/*",
```

Comment:
do we still need this？

Reply:
I think the new `kbn-plugin-helper` can include the target in the artifact, so no need to specify here. I verified with reporting. You can also give it a try

Reply:
you are right it doesn't matter now, removed